gesture/gesture.go

```diff
@@ -37,6 +37,8 @@ type Click struct {
 	clicks int
 	// pressed tracks whether the pointer is pressed.
 	pressed bool
+	// Buttons tracks which buttons are pressed.
```

Comment:
This doesn't seem quite right. Did you mean

// Buttons specify the set of buttons to track. The zero value is equivalent to the
// primary button, not the empty set.

?

Ah, I see from the rest of your change that I wasn't clear enough in my description. What I meant by the Buttons field is that it should be set by the *user* from Click, not Click itself. The purpose of (the zero value of) Buttons is maintain the old behaviour where only the primary mouse button would be tracked. Users such as yourself that need to track other buttons can include them in Buttons.